inclusive-design/wecount.inclusivedesign.ca
# fix: "npm run start" no longer needs to run twice to start the dev site (resolves #298)

## Description

WeCount website will start correctly with the first run of "npm run start".

## Steps to test

1. Run "npm run start" at a command line interface;
2. Go to "http://localhost:3000";
3. The WeCount website should load and work correctly;
4. Go to the source code, change any file and save;
5. At the command line interface where "npm run start" runs, the site should be rebuilt;
6. Go back to "http://localhost:3000";
7. The change should be loaded;
8. Go to the command line interface and terminates "npm run start" using CTRL + C;
9. Issue these commands: "npm run build; npx serve dist";
10. Go to "http://localhost:5000";
11. The WeCount website should load and work correctly.

**Expected behavior:** 

See above.

## Additional information

The issue was because with the previous implementation, the webpack `laravel-mix` creates a new directory "/src/_include/static" with static contents that should be copied to "/dist" by the Eleventy. However, Eleventy config steps run too fast and the copy commands happen before "/src/_include/static" is generated.

The solution is to use Eleventy features to handle tasks performed by `laravel-mix`.
